Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM node:10
+ENV PRIVATE_KEY key.pem
```

Comment:
where do these key/certificates come from for another use, needs to be documetned
also the whole docker bit needs to be documented in the readme